src/main/java/edition/academy/seventh/database/connector/PostgresConnector.java

```diff
@@ -22,6 +23,12 @@
     settings.put("hibernate.dialect", "org.hibernate.dialect.PostgreSQL10Dialect");
     settings.put("hibernate.show_sql", "true");
     settings.put("hibernate.hbm2ddl.auto", "update");
+
     return settings;
   }
+
+  @Override
+  public EntityManagerFactory getEntityManagerFactory() {
```

Comment:
czemu to nadpisuemy z super klasy, jak nic nie zmieniamy?
w H@Connector nie ma tej metody

Reply:
Poprawione :)